src/renderer/helpers/fp/from.ts

```diff
@@ -0,0 +1,3 @@
+import * as O from 'fp-ts/Option'
+
+export const optionFromNullableString = O.fromPredicate<string | undefined | null, string>((s): s is string => !!s)
```

Comment:
[nitpick] `optionFromNullableString` -> `optionFromString`

Reply:
actually initially i set name to `optionFromString` but then i've added `nullable` to it 'cuz this function creates value from nullable values too. what do you think ?

Reply:
Both names are valid. It was just a thought about this name, because we still have types to describe what a function expects as parameters. Anyway, I'm fine with both names :smile: